src/routes/safe/components/Apps/components/AppFrame.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen, fireEvent, within, act, waitFor } from 'src/utils/test-utils'
+import AppFrame from './AppFrame'
+import axios from 'axios'
+import * as safeAppsUtils from 'src/routes/safe/components/Apps/utils'
+
+jest.mock('axios')
+
+describe('Safe Apps -> AppFrame', () => {
+  beforeEach(() => {
+    // @ts-ignore
+    safeAppsUtils.canLoadAppImage = jest.fn().mockResolvedValue(true)
+    // @ts-ignore
+    axios.get.mockImplementation((url: string) => {
+      console.log(url)
```

Comment:
console.log sneaked in

Reply:
done!. Yep, i will wait to Dani's to review it